book/Makefile

```diff
@@ -26,4 +26,4 @@ pathways:
 	pathways ./website/
 
 clean:
-	jupyter-book clean -a ./website
+	jupyter-book clean --all --yes ./website/_build
```

Comment:
What does the `--yes` flag do? I can't see it in the JB documentation?
And what is the impact of manually listing the path at the end?

Meta-comment: should we add comments into this file as to what the impact/purpose of the flags are, for quick reference? It would make it slightly less clean, but perhaps easier to understand if anyone is looking at this file as a reference for what to do on a different project.